Cubical/HITs/ListedFiniteSet/Base.agda

```diff
@@ -89,3 +91,27 @@ module PropElim {ℓ}
     (λ _ _ _ → isOfHLevel→isOfHLevelDep 1 trunc* _ _ _)
     (λ _ _ → isOfHLevel→isOfHLevelDep 1 trunc* _ _ _)
     λ xs → isProp→isSet (trunc* xs)
+
+_++_ : ∀ (xs ys : LFSet A) → LFSet A
+[]                  ++ ys = ys
+(x ∷ xs)            ++ ys = x ∷ (xs ++ ys)
+dup x xs i          ++ ys = dup x (xs ++ ys) i
+comm x y xs i       ++ ys = comm x y (xs ++ ys) i
+trunc xs zs p q i j ++ ys = trunc
+  (xs ++ ys) (zs ++ ys) (cong (_++ ys) p) (cong (_++ ys) q) i j
+
+map : (A → B) → LFSet A → LFSet B
+map f [] = []
+map f (x ∷ xs) = f x ∷ map f xs
+map f (dup x xs i) = dup (f x) (map f xs) i
+map f (comm x y xs i) = comm (f x) (f y) (map f xs) i
+map f (trunc xs ys p q i j) = trunc
```

Comment:
Indentation is weird here, why not put the `trunc` with all of its arguments on the next line instead of splitting it over 6 lines?

Reply:
Similar indentation occurs in the Properties file

Reply:
In this case it was just because I find it more legible. In the properties file I exceed 100 characters if all the arguments go in the same line (I find 80 characters too long). I have joined some arguments in the same line though.

Agda needs a standard style so bad...

Reply:
Ok, it looks better now!